packages/orm/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "experimentalDecorators": true,
     "allowJs": true,
-    "declaration": true,
+    "declaration": false,
```

Comment:
I set a declaration prop to false for now. TS throws me an error whenever I export functions that create models (has to be that way because of decorators + global registry) and use the types of those models somewhere outside of that function.

```ts
export const createModels = () => {
  // Should be exported but it is impossible to achieve inside a fn block
  class Test extends Model<...> {}

  return { Test };
}

// Since I am using the class as a type here, I need to export the class as well. It is needed to create declaration files.
export type SomeType = {
  Test: ReturnType<typeof createModels>['Test'];
}

Reply:
I think I will need to extend the tsconfig to not use that option in test files or something.